lit-base/lambda_function.py

```diff
@@ -0,0 +1,19 @@
+from model_service import preprocess, inference, postprocess
+
+
+def lambda_handler(event, context):
+
+    # serving 
+    input    = preprocess(event)
+    output   = inference(input)
+    response = postprocess(output)
+    
+    out = {
```

Comment:
create a class with name 'response' & define this response there - this will allow for extending/modifying the response as needed in the future